### ISSUE TITLE
"Undo" message doesn't resize if text is longer

### ISSUE THREAD

**Issue by Chartman123** [nextcloud/mail#6364]
### Steps to reproduce

1. Send an email with e.g. German user language
2. See "undo" message is not really readable

### Expected behavior

Message popup should resize to the text length + button width

### Actual behavior

Message popup seems to have a fixed with:

![image](https://user-images.githubusercontent.com/16020496/166879915-117f07d1-7121-498a-922e-ea5e8788bdc6.png)

### Mail app version

1.12.0

### PHP engine version

PHP 8.0

### Web server

Apache (supported)

### Database

PostgreSQL

**Comment by ChristophWurst** [comment]
Possibly a bug of https://github.com/nextcloud/nextcloud-dialogs/ because I don't think this app influences how the text is styled

**Comment by Chartman123** [comment]
Yes, possible. Should I open a bug report over there? The problem here was the first time I was facing this issue...

**Comment by ChristophWurst** [comment]
One of us will have a look, no worries. If it's a bug there then we can create another ticket. For now I'd leave it here until my suspicion has been verified.

**Comment by Chartman123** [comment]
I had a closer look at the code. I think that it is indeed a problem in the dialogs repo... The undo button will be added to the toast message there... Seems like the space reserved for the button stays in the size of the "Undo" text even if there is a longer text ("Rückgängig") so the toast message isn't correctly resized to the bigger button.

https://github.com/nextcloud/nextcloud-dialogs/blob/3ca4519d6763249238b82a9f6135c87f30c86937/styles/toast.scss#L38
https://github.com/nextcloud/nextcloud-dialogs/blob/3ca4519d6763249238b82a9f6135c87f30c86937/lib/toast.ts#L189-L194

**Comment by GretaD** [comment]
> I had a closer look at the code. I think that it is indeed a problem in the dialogs repo... The undo button will be added to the toast message there... Seems like the space reserved for the button stays in the size of the "Undo" text even if there is a longer text ("Rückgängig") so the toast message isn't correctly resized to the bigger button.
> 
> https://github.com/nextcloud/nextcloud-dialogs/blob/3ca4519d6763249238b82a9f6135c87f30c86937/styles/toast.scss#L38
> 
> https://github.com/nextcloud/nextcloud-dialogs/blob/3ca4519d6763249238b82a9f6135c87f30c86937/lib/toast.ts#L189-L194

thank you so much for investigating,  i will fix this asap
